lua/lazyvim/plugins/extras/lang/graphql.lua

```diff
@@ -0,0 +1,35 @@
+return {
+  -- add graphql to treesitter
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "graphql" })
+      end
+    end,
+  },
+
+  -- correctly install lsp with mason
+  {
+    "williamboman/mason.nvim",
+    opts = function(_, opts)
+      vim.list_extend(opts.ensure_installed, {
+        "graphql-language-service-cli",
+      })
+    end,
+  },
```

Comment:
```suggestion
```
This isn't needed, `mason` installs the server already, since you put `graphql` in the `nvim-lspconfig` opts.
Here is my working config:
https://github.com/Jomik/dotfiles/blob/7f709d13ae9676c76fb0ab5fa479001358ae7140/.config/nvim/lua/plugins/lang/graphql.lua#L14

Reply:
This plugin is meant to fully work without you having to add anything to your config other than the plugin line in `lazy.lua`.

Reply:
Yup.. There is no need to do anything more. `mason` automatically installs servers added in `nvim-lspconfig`.
That is done in all the other language plugins as well.
Check the json or go plugins for further examples.
https://github.com/LazyVim/LazyVim/blob/274649e92321d615888f889476c9329f8d831e8e/lua/lazyvim/plugins/extras/lang/json.lua#L23 
You can also simply try without and see if it isn't automatically installing it for you already 👍

Reply:
It does appear to still install when dropping the ensure install. Thanks